applications/FluidDynamicsApplication/python_scripts/adjoint_fluid_solver.py

```diff
@@ -0,0 +1,220 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
It seems like this file copies many functionalities from the fluids solver

Maybe it would make sense to derive from it? @jcotela

Reply:
I agree, but maybe not in this PR.